.docker/images/nginx/helpers/230-favicon.conf

```
@@ -0,0 +1,4 @@
+# Try to find a favicon in the app root otherwise default to govcms' icon.
+location /favicion.ico {
+  try_files $uri /profiles/contrib/govcms/favicon.ico
```

```suggestion
  try_files $uri /profiles/contrib/govcms/favicon.ico;
```
fixes `nginx: [emerg] unexpected "}" in /etc/nginx/helpers/230-favicon.conf:4` error